REPL.cpp

```diff
@@ -1,7 +1,19 @@
 #include "REPL.h"
+#include "TransformAST.h"
 
 #define SWIFT_MODULE_PATH "S:\\b\\swift\\lib\\swift\\windows\\x86_64"
 
+void ConfigureFunctionLinkage(std::unique_ptr<swift::SILModule> &sil_module)
+{
+    for(auto &fn : sil_module->getFunctionList())
+    {
+        fn.setLinkage(swift::SILLinkage::Public);
+        std::cout << "Set SIL Function " << fn.getName().str() << " to public\n";
+    }
+
+    sil_module->lookUpFunction("main")->setLinkage(swift::SILLinkage::Private);
```

Comment:
Why is main private?

Reply:
We make the SourceFile have type Main, so it'll generate a main function in the LLVM IR. We don't want one, so we make it private so that it doesn't get generated. Instead of having main, we'll JIT a different function and call that.